src/porepy/numerics/fv/biot.py

```diff
@@ -299,20 +300,26 @@ def discretize(self, sd: pp.Grid, sd_data: dict) -> None:
                     Defaults to 1.
 
             Related to numerics:
-                inverter: Which method to use for block inversion. See
+                - inverter (``str``): Which method to use for block inversion. See
                     pp.fvutils.invert_diagonal_blocks for detail, and for default
                     options.
-                mpfa_eta: Location of continuity point in MPSA. Defaults to 1/3 for
-                    simplex grids, 0 otherwise.
-
-        The discretization is stored in the data dictionary, in the form of
-        several matrices representing different coupling terms. For details,
-        and how to combine these, see self.assemble_matrix()
+                - mpfa_eta (``float``): Location of continuity point in MPSA. Defaults
```

Comment:
The combination of "mpfa_eta" and "partition_arguments" does not feel very "natural". I suggest we keep for now, but keep the question of how to specify discretization parameters in mind.

Reply:
I agree they are diverse, but they both relate to choices in the numerical method. I agree, let's keep it for now.